Minor improvements

moraisandre opened the pull request:
Hey there!

I just made some minor improvements like adding 'Main + Extras' time. I also added Time Labels to distinguish what the times attributes represented (e.g. The attribute gameplayMain could be 'Solo' or 'Main Story') and updated the README file.

Patched bugs:
* On detail method in some cases the value returned NaN instead of 0
* Solo, Co-Op & Vs. Times aren't fetched 

Tks for developing this package mate!

ckatzorke commented:
Hi André - thanks for the contribution!!
I never expected that this package gets any reception at all :-)
Thanks for adding the distinction between solo and multiplayer (I only used Solo) and adding mainExtra. I never used it, since it was often empty, and I interpolated - for a smal hobby project for managing your pile of shame piletron3000.katzorke.io (caution- WIP ;-) )
Thanks - only one small remark (let over var)- and could you add a test for the multi/solo distinction?
Thx
Christian

moraisandre commented:
Hey Christian,

I added 2 new tests and updated the others ones to include 'Main + Extra' time. I also replaced the var to let.

I hope you like it!

Regards,
André Morais
